packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php

```diff
@@ -0,0 +1,110 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Laravel\Rector\StaticCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\BinaryOp\Mul;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Scalar\LNumber;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+/**
+ * @see https://github.com/laravel/framework/pull/27276
+ */
+final class MinutesToSecondsInCacheRector extends AbstractRector
+{
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition(
+            'Change minutes argument to seconds in Illuminate\Contracts\Cache\Store and Illuminate\Support\Facades',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run()
+    {
+        Illuminate\Support\Facades::put('key', 'value', 60);
+    }
+}
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run()
+    {
+        Illuminate\Support\Facades::put('key', 'value', 3600);
```

Comment:
You actually modifying it to 60*60, right?

Reply:
Yes, later I conclude it's more readable. 

I forgot to fix it here, thanks :+1: